lib/src/test/java/com/auth0/android/lock/WebProviderTest.java

```diff
@@ -40,6 +43,9 @@
     @Rule
     public ExpectedException exception = ExpectedException.none();
 
+    @Mock
+    public Map<String, Object> map;
```

Comment:
used only once in https://github.com/auth0/Lock.Android/pull/410/files#diff-206bfaf63e03ad1b5fec7bc2d3b5005bR61
since you're not checking anything.. can't you use null like in the other places?